packages/core-mobile/app/seedless/services/SeedlessExportService.ts

```diff
@@ -1,19 +1,21 @@
 import {
   CubeSignerResponse,
-  MemorySessionStorage,
+  MemorySessionManager,
   MfaReceipt,
-  SignerSessionData,
+  SessionData,
   UserExportCompleteResponse,
   UserExportInitResponse,
   userExportDecrypt,
   userExportKeygen
 } from '@cubist-labs/cubesigner-sdk'
-import SeedlessSessionManager from './SeedlessSessionManager'
+import SeedlessSession from './SeedlessSession'
 
 class SeedlessExportService {
-  sessionManager = new SeedlessSessionManager({
+  session = new SeedlessSession({
     scopes: ['export:user', 'manage:export:user', 'manage:mfa:vote'],
-    sessionStorage: new MemorySessionStorage<SignerSessionData>()
+    // passing null as session data initially
+    // user will perform OIDC auth when the export flow starts
+    sessionManager: new MemorySessionManager(null as unknown as SessionData)
```

Comment:
the old MemorySessionStorage doesn't require passing session data to its constructor but the new one MemorySessionManager does. doing null here since we want to separate the 2 sessions (app session and export session).